src/test/java/uk/gov/pay/webhooks/message/dao/WebhookMessageDaoTest.java

```diff
@@ -40,6 +41,32 @@ public void setUp() {
         webhookDeliveryQueueDao = new WebhookDeliveryQueueDao(database.getSessionFactory(), InstantSource.fixed(Instant.now()));
     }
 
+    @Test
+    public void shouldDeleteWebhookMessages() {
+        database.inTransaction(() -> {
+            var webhook = new WebhookEntity();
+            var webhookMessage1 = createWebhookMessageEntity(webhook);
+            var webhookMessage2 = createWebhookMessageEntity(webhook);
+            var webhookMessage3 = createWebhookMessageEntity(webhook);
+            webhookMessageDao.deleteMessages(Stream.of(webhookMessage1, webhookMessage2, webhookMessage3));
+        });
+    }
+
+    private WebhookMessageEntity createWebhookMessageEntity(WebhookEntity webhook) {
```

Comment:
maybe move this down to the other private methods

Reply:
fixed